ai_data_preprocessing_queue/Steps/regex_replacement.py

```diff
@@ -38,4 +40,4 @@ def _prepare_pre_processor_data(preprocessorData: str) -> pandas.DataFrame:
     csv["sort"] = csv[2]
 
     # sort
-    return csv.sort_values("sort", inplace=False).drop("sort", "columns")
+    return csv.sort_values("sort", inplace=False).drop("sort", axis=1)
```

Comment:
this was the only breaking change. We already got a deprecation warning before the update. See task for a link to the related github issue for this deprecation